erigon-lib/state/aggregator.go

```diff
@@ -370,6 +369,24 @@ func (a *Aggregator) LS() {
 	}
 }
 
+func (a *Aggregator) WaitForBuildAndMerge(ctx context.Context) chan struct{} {
```

Comment:
is this used anywhere?

Reply:
In stage_custom_trace

Reply:
are you planning to add it later, or it should be there in this pr? This pr doesn't have it.